disputes.go

```diff
@@ -40,8 +40,11 @@ type Dispute struct {
 	Fields map[string]interface{} `json:"fields"`
 	// Any fields required by the template that have not yet been provided.
 	MissingFields map[string]interface{} `json:"missing_fields"`
-	// A list of products in the disputed order. (See [Product data](#product-data) for details.) (optional)
+	// A list of products in the disputed order. (See [Product data](https://www.chargehound.com/docs/api/index.html#product-data) for details.) (optional)
 	Products []Product `json:"products"`
+	// List of emails with the customer.
+	// (See [Customer correspondence](https://www.chargehound.com/docs/api/index.html#customer-correspondence) for details.) (optional)
+	Correspondence []CorrespondenceItem `json:"correspondence,omitempty"`
```

Comment:
This is omitempty but products is not. Perhaps they should have similar behavior